api/pkg/cli/cmd/install/install.go

```diff
@@ -153,6 +154,15 @@ func (opts *options) run() error {
 		}
 	}
 
+	var deprecated bool
+	if opts.resource.GetAnnotations()[deprecationAnnotation] == "true" {
+		deprecated = true
```

Comment:
Are we using the variable anywhere again, if not can't we directly log here ??

Reply:
no,it needs a boolean

Reply:
I think this should work 

```
	if opts.resource.GetAnnotations()[deprecationAnnotation] == "true" {
		_ = printer.New(out).String("WARN: this version has been deprecated")
	}
```